src/librustc/mir/mod.rs

```diff
@@ -2827,7 +2830,7 @@ impl Location {
 #[derive(Copy, Clone, Debug, PartialEq, Eq, Hash, RustcEncodable, RustcDecodable, HashStable)]
 pub enum UnsafetyViolationKind {
     General,
-    /// Permitted in const fn and regular fns.
+    /// Permitted in const fns and regular fns.
```

Comment:
```suggestion
    /// Permitted in `const fn`s and regular `fn`s.
```

Reply:
I think it's fine as is, because "fn" is used all over the code simply as an abbreviation for "function", not necessarily the syntax.

Reply:
I've done changes/reviewed things re. `UnsafetyViolationKind` so know that this is about the syntax in this context.

Reply:
Okay, sure.